src/lib/auth.ts

```diff
@@ -86,15 +80,6 @@ export interface TeacherUser extends User {
   networks?: string[];  // list of networks available to teacher
 }
 
-export interface RawClassInfo {
-  uri: string;
-  name: string;
-  state: string;
-  class_hash: string;
-  teachers: RawUser[];
-  students: RawUser[];
-}
-
 export interface ClassInfo {
```

Comment:
I couldn't help noticing that most of the code for this project uses an 'I' prefix, for TypeScript interfaces, but not ALL interface declarations (such as this one). Do we have a consensus on the naming of interfaces. (FWIW, in my experience there's no agreement, industry wide, on whether to use the 'I' prefix, or not. I'm happy to adhere to either convention; but preferably do it consistently...)

Reply:
It's a matter of opinion. I tend to like it most of the time, but I don't feel terribly strongly about it and other engineers sometimes prefer it without. So I generally use it when I'm defining new interfaces but won't necessarily change existing code if there isn't another reason to do so.